deploy.rb

```diff
@@ -64,6 +76,7 @@ def build_nightly
     reset_nightly
 
     Dir.chdir(BUILD_DIR + '/nightly') do
+      build_pr_branch('master')
```

Comment:
Minor but maybe move the `Building Nightly` message above below this line so that the output would read:

```
Testing PR 77 against branch master
Building nightly...
Building version 2.0
Copying 2.0
Building version 2.1
Copying 2.1
```